### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by fivetran-jamie**
- move non-hubspot specific macros to utils
- update dbt version
- update dbt utils

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 packages:
   - package: fishtown-analytics/dbt_utils
-    version: [">=0.4.0"]
+    version: [">=0.6.0", "<0.7.0"]
   
   - package: fivetran/hubspot_source
-    version: 0.1.0
\ No newline at end of file
+    version: 2.0.0
```
Comment (fivetran-jamie) [inline packages.yml:6]: this should probably be [">=2.0.0", "<3.0.0"] instead right?

**Comment by fivetran-kristin** [comment]
@fivetran-jamie I left some comments and also can confirm that I've ran the latest Hubspot package on the data and it runs without error.

File: packages.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 packages:
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply packages.yml:1]: ah good point!

**Review by fivetran-kristin — APPROVED** [review]
LGTM

**Pull request merged**